lib/analyzer/applications/node-modules-utils.ts

```diff
@@ -206,6 +208,17 @@ function groupFilesByDirectory(
   return filesByDir;
 }
 
+function isNodeAppFile(filepath: string): boolean {
```

Comment:
It might be better to have this as a part of the `getNodeJsTsAppFileContentAction`.